andela/mmdp_site
# MMDP-500 Implement manage registration status dropdown

**What does this PR do?**

This PR implements CRUD operations for the registration status dropdown

**Description of Task to be completed?**

As an administrator, I want to be able to manage options in the 'Registration Status' drop down

**How should this be manually tested?**

Git clone https://github.com/andela/mmdp_site
Check out  ft-registration-status-MMDP-500 branch using git.
Run yarn run: server to start the server.

**Any background context you want to provide?**

None

**What are the relevant Jira issues?**

[MMDP-500](https://viisaus.atlassian.net/browse/MMDP-500)
